sample-apps/auto-instrumentation-java-spark/src/main/java/com/amazon/sampleapp/MetricEmitter.java

```diff
@@ -0,0 +1,232 @@
+package com.amazon.sampleapp;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleHistogram;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileNotFoundException;
+
+public class MetricEmitter {
+  private static final Logger logger = LogManager.getLogger();
+  private static Config getConfig() { return new Config(); }
+  static final AttributeKey<String> DIMENSION_API_NAME = AttributeKey.stringKey("apiName");
+  static final AttributeKey<String> DIMENSION_STATUS_CODE = AttributeKey.stringKey("statusCode");
```

Comment:
These metric attribute names are outdated. The new ones are following this key-value convention 
{
“signal”:     (string)“metric”
“language”:   (string)<name of language used>
“metricType”: (string)“random” or (string)"request"
}